google/resource_container_cluster.go

```diff
@@ -279,6 +279,48 @@ func resourceContainerCluster() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"ip_allocation_policy": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"use_ip_aliases": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Computed: true,
+							Default:  nil,
+							ForceNew: true,
+						},
+						"cluster_secondary_range_name": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ForceNew: true,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								if new == "" {
+									return true
+								}
+								return ipCidrRangeDiffSuppress(k, old, new, d)
```

Comment:
Isn't this field the name, not the range itself?

Reply:
🤦‍♂️